addons/website_blog/views/website_blog_templates.xml

```diff
@@ -487,5 +491,55 @@
          <li><a href="#" data-action="new_blog_post">New Blog Post</a></li>
     </xpath>
 </template>
+
+<!-- RSS Feed -->
+<template id="website_blog_rss" name="Blog RSS">&lt;?xml version="1.0" encoding="utf-8"?&gt;
+    <rss version="2.0" xmlns:dc="http://purl.org/dc/elements/1.1/">
+        <channel>
+            <title><t t-esc="blog.name" /></title>
+            <link><t t-esc="'%s/blog/%s' % (base_url, blog.id)"/></link>
+            <description><t t-esc="blog.name" /></description>
+            <docs>http://www.rssboard.org/rss-specification</docs>
+            <lastBuildDate><t t-esc="(blog.formate_datetime_zone(blog.write_date)).strftime('%a, %d %b %Y %H:%M:%S %z')" /></lastBuildDate>
+            <item t-foreach="blog_posts" t-as="post">
+                <title><t t-esc="post.name" /></title>
+                <link><t t-esc="base_url + post.website_url"/></link>
+                <description><t t-esc="post.subtitle" /></description>
+                <author>
+                    <t t-esc="post.author_id.email" /> (<t t-esc="post.author_id.name" />)
+                </author>
+                <pubDate><t t-esc="(blog.formate_datetime_zone(post.create_date)).strftime('%a, %d %b %Y %H:%M:%S %z')" /></pubDate>
+                    <guid><t t-esc="base_url + post.website_url"/></guid>
+            </item>
+        </channel>
+    </rss>
+</template>
+    
+<!-- ATOM Feed -->
+<template id="website_blog_atom" name="Blog ATOM">&lt;?xml version="1.0" encoding="utf-8"?&gt;
+    <feed t-att-xmlns="'http://www.w3.org/2005/Atom'">
+        <id><t t-esc="blog_url"/></id>
+        <title><t t-esc="blog.name"/></title>
+        <link type="text/html" rel="alternate" t-att-href="blog_url"/>
+        <link type="application/atom+xml" rel="self" t-att-href="'%s/atom' % (blog_url)"/>
+        <subtitle><t t-esc="blog.subtitle"/></subtitle>
+        <updated><t t-esc="(blog.formate_datetime_zone(blog.write_date)).isoformat('T')" /></updated>
```

Comment:
can't you do this formatting in the controller? (eventually add a parameter to formate_datetime_zone to specify format)